### PR TITLE
Split levels of verbosity of test logging on components.

### PR DESCRIPTION
**Pull request by gmarek**
Needed for #19048

cc @zmerlynn @roberthbailey @fgrzadkowski

### CODE REVIEW

File: cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest [hunk 1]
```diff
@@ -30,7 +30,7 @@
   {% if grains.cloud != 'vagrant' -%}
     {% set cloud_provider = "--cloud-provider=" + grains.cloud -%}
   {% endif -%}
-  {% set service_account_key = " --service-account-private-key-file=/srv/kubernetes/server.key " -%}
+  {% set service_account_key = "--service-account-private-key-file=/srv/kubernetes/server.key" -%}
```
Comment (wojtek-t) [inline cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest]: why this change?

Comment (wojtek-t) [inline cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest]: ok - seems that consistency

**Comment by wojtek-t** [comment]
LGTM - thanks!

**Comment by gmarek** [comment]
@k8s-bot unit test this

**Comment by gmarek** [comment]
`ERROR: Error fetching remote repo 'origin'`

@k8s-bot unit test this

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please
